src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -624,13 +701,47 @@ internal bool IsAutoPropertyWithGetAccessor
             => IsAutoProperty && _getMethod is object;
 
         protected bool IsAutoProperty
-            => (_propertyFlags & Flags.IsAutoProperty) != 0;
+        {
+            get
+            {
+                EnsureAccessorsBinding();
+                return (_propertyFlags & Flags.IsAutoProperty) != 0;
+            }
+        }
+
+        private protected bool IsAutoPropertyWithoutAccessorBinding
```

Comment:
>IsAutoPropertyWithoutAccessorBinding [](http://example.com/codeflow?start=31&length=36)

It doesn't look like this property is necessary, especially that it changes its value during life-time of the object and stops returning accurate information after the flag is modified.